.travis.yml

```diff
@@ -1,8 +1,32 @@
+conditions: v1
 language: python
-sudo: required
-dist: xenial
-python: "3.7"
-cache: pip
+python:
+  - "3.6"
```

Comment:
Also, it's usually a good idea to test against other operating systems (macOS/Windows).
Ref: https://docs.travis-ci.com/user/languages/python/#running-python-tests-on-multiple-operating-systems